### PR TITLE
Dockerfile

### LINKED ISSUE #4: Container Docker w/ WordPress
- Containerize the application using Docker (see Resources section).

### PR DESCRIPTION
**Pull request by faria-marcio**
Dockerfile with Linux, Apache and WordPress for our project.

### CODE REVIEW

**Comment by starkmatt** [comment]
Hey Marcio,
LGTM,

I was able to build the wordpress container, although I wasn't able to test navigate to it.
Perhaps once we have the DB Solution up we will be able to create a POC in AWS to see how it behaves

**Review by starkmatt — DISMISSED** [review]
LGTM

**Comment by starkmatt** [comment]
Hey Marcio,
I believe the next step for us is to get our docker container within the ECR.
And connect our docker wp instance to our DB so we can test the connection

File: project1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#Wordpress docker image with Php 7.4 and Apache
+FROM wordpress:php7.4-apache
+LABEL Maintainer = "Marcio Faria <marciodefaria@gmail.com>" \
+      Description = "LAP stack to run WordPress."
+
+# wp_data volume
+VOLUME /var/www/html
+WORKDIR /var/www/html
+
```
Comment (starkmatt) [inline project1/Dockerfile]: I say let's test these environment variables with our DB Instance
he following environment variables are also honored for configuring your WordPress instance:

-e WORDPRESS_DB_HOST=...
-e WORDPRESS_DB_USER=...
-e WORDPRESS_DB_PASSWORD=...
-e WORDPRESS_DB_NAME=...
-e WORDPRESS_TABLE_PREFIX=...

https://hub.docker.com/_/wordpress/

**Comment by starkmatt** [comment]
https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-push-ecr-image.html

**Review by deniseddi — DISMISSED** [review]
Approving it knowing that there might be a few alterations later on based on the comments:
 "I'll update it once ew have RDS database defined."
"You guys can look at docker compose to make easier for both containers to communicate locally"

File: project1/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#Wordpress docker image with Php 7.4 and Apache
```
Comment (kikobr82) [inline project1/Dockerfile]: Move this file to a folder that reflects what the content is related.

maybe `wordpress`...

File: project1/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+FROM wordpress:php7.4-apache
+LABEL Maintainer = "Marcio Faria <marciodefaria@gmail.com>" \
+      Description = "LAP stack to run WordPress."
```
Comment (kikobr82) [inline project1/Dockerfile]: you can remove those lines

**Review by kikobr82 — CHANGES_REQUESTED** [review]
take a look at the coments.

File: project1/Dockerfile (hunk 2 above)
Comment (faria-marcio) [reply project1/Dockerfile]: Done

File: project1/Dockerfile (hunk 3 above)
Comment (faria-marcio) [reply project1/Dockerfile]: Done

File: project1/Dockerfile (hunk 1 above)
Comment (faria-marcio) [reply project1/Dockerfile]: Done

**Review by faria-marcio — COMMENTED** [review]
I've done the changes requested/suggested. Ready to merge with master.

**Review by kikobr82 — APPROVED** [review]
lgtm

**Pull request merged**